woocommerce/woocommerce-ios
# [My Store Updates] Header UI updates

Part of: #5743 


### Description


There are a few design updates on the store stats header for Home Screen M2, the design link and p2 post is p91TBi-6Of-p2. Unfortunately, it is not easy to convert the header UI to SwiftUI because of the existing xib setup and Jetpack empty state UI. Since it's hard to review diffs on the xib file, I separated the commits for each change.

I plan to do a followup refactoring to separate the stats values to a view model, so that we can unit test the non-trivial value generation as text.

### Testing instructions


- Launch the app
- In the My store tab, tap each time range tab --> the stats values should look as before, with a new "conversion" value that is `order count/visitor count` and max 100% (my test sites all have 100% 😅 )
- Feel free to try switching to dark/light mode (there is still some work left to fully support dynamic type, but it's tricky since we currently have to manually set the view height constraint probably because Auto Layout limitations)

@jaclync tests:
- [x] JCP site
- [x] Feature flag disabled: the previous stats UI should be shown
- [x] UI tests pass ([CI workflow](https://app.circleci.com/pipelines/github/woocommerce/woocommerce-ios/16441/workflows/8c167b3e-1d62-4658-bb64-4f70337aac33))

### Screenshots

\ | JCP site | today | this week | this month | this month - highlighted | this year | this year - highlighted
-- | -- | -- | -- | -- | -- | -- | --
dark | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 11 10](https://user-images.githubusercontent.com/1945542/147714157-55aeb189-f3a2-4171-8186-7daad1a54b53.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 16 09](https://user-images.githubusercontent.com/1945542/147714160-a7959458-3854-4398-a0d7-13a38205b5b5.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 16 11](https://user-images.githubusercontent.com/1945542/147714162-4b7bf87f-4f6a-41e6-bb6c-0f1b0e8d8e1d.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 16 14](https://user-images.githubusercontent.com/1945542/147714164-c5ccb786-5718-416b-9027-5b5877e3bf10.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 16 16](https://user-images.githubusercontent.com/1945542/147714165-20401212-2b1f-4fe5-9327-09441b7878ea.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 16 22](https://user-images.githubusercontent.com/1945542/147714166-9f94bdf7-1cb8-4b0c-bf88-4d1871638f2d.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 16 24](https://user-images.githubusercontent.com/1945542/147714168-83e315ef-34dc-4260-a6cb-16252db71fe4.png)
light | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 11 20](https://user-images.githubusercontent.com/1945542/147714224-08d43278-8a97-4864-a6ea-dbdee1f783e0.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 15 47](https://user-images.githubusercontent.com/1945542/147714227-4a7c8983-a783-4d25-83b0-cd28d9284a37.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 15 48](https://user-images.githubusercontent.com/1945542/147714228-d3b482cb-5579-4334-9202-f91a68ab5d8f.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 15 51](https://user-images.githubusercontent.com/1945542/147714230-da5926ce-1203-4b41-83d9-ed71589f5fd4.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 15 55](https://user-images.githubusercontent.com/1945542/147714231-73b20d2c-31c7-4bf8-8a62-4a07698089e6.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 15 57](https://user-images.githubusercontent.com/1945542/147714233-daf341b2-afc9-4265-a072-35c6af5da6e9.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-30 at 09 16 01](https://user-images.githubusercontent.com/1945542/147714235-ebd953ad-b1a0-46fc-8e32-45cd6d7e0331.png)

---
